runtime/suspend-transform-runtime/src/jsMain/kotlin/love.forte.plugin.suspendtrans/runtime/RunInSuspendJs.kt

```diff
@@ -5,8 +5,9 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.promise
 import kotlin.coroutines.CoroutineContext
 import kotlin.js.Promise
+import kotlinx.coroutines.SupervisorJob
 
-private val CoroutineContext4Js: CoroutineContext = Dispatchers.Default
+private val CoroutineContext4Js: CoroutineContext = Dispatchers.Default + SupervisorJob()
```

Comment:
To be discussed: is there any difference between using a custom scope and `GlobalScope`?

Reply:
Okay. The merged version has been released and you can try to update it now!